pact/concrete-policies/collection-policy/collection-policy-v1.pact

```diff
@@ -58,15 +58,18 @@
       \ * collection_id:string - registers the token to a collection and emits           \
       \ TOKEN_COLLECTION event for discovery"
       (enforce (>= collection-size 0) "Collection size must be positive")
-      (let ((collection-id:string (create-collection-id collection-name) ))
+      (let ((collection-id:string (create-collection-id collection-name operator-guard) ))
         (insert collections collection-id {
          "id": collection-id
          ,"name": collection-name
          ,"max-size": collection-size
          ,"size": 0
          ,"operator-guard": operator-guard
         })
-        (emit-event (COLLECTION collection-id collection-name collection-size))
+        ;  enforce operator guard and emit collection created event
+        (with-capability (OPERATOR collection-id)
```

Comment:
I think we can place this in L60, as we don the insert call without OPERATOR cap :)

Reply:
The cap reads the guard from the collections table, if we put the cap there it will fail. This is merely a sanity check that safeguards that the user is indeed able to sign the guard that he provides